scripts/cleanup_dp.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+
+mkdir ~/.aws; touch ~/.aws/config
+
+echo "[default]
+aws_access_key_id = $AWS_ACCESS_KEY_ID
+aws_secret_access_key = $AWS_SECRET_ACCESS_KEY" > ~/.aws/credentials
+
+echo "[default]
+[profile eksci]
+role_arn= $AWS_ROLE_ARN
+output = json
+region=ap-south-1
+source_profile = default" > ~/.aws/config
+
+export region=ap-south-1
+export cluster_name=uat-cluster
+
+echo "Region: $region"
+echo "Cluster name: $cluster_name"
+
+sts_output=$(aws sts assume-role --role-arn env.AWS_ROLE_ARN --role-session-name ekscisession)
+export AWS_ACCESS_KEY_ID=$(echo $sts_output | jq -r '.Credentials''.AccessKeyId');\
+export AWS_SECRET_ACCESS_KEY=$(echo $sts_output | jq -r '.Credentials''.SecretAccessKey');\
+export AWS_SESSION_TOKEN=$(echo $sts_output | jq -r '.Credentials''.SessionToken');
+
+aws eks update-kubeconfig --region $region --name $cluster_name --profile eksci
```

Comment:
Now this should work without depending on anything in the `~/.aws` folder.

```suggestion
aws eks update-kubeconfig --name "$cluster_name" --role-arn "$AWS_ROLE_ARN"
```